src/components/books/IndividualBook.js

```diff
@@ -1,46 +1,31 @@
+import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import PropTypes from 'prop-types';
```

Comment:
- [ ] As required by Microverse, you are not meant to use any linters disable rule in your code base. The linters are there to make sure your code follows the best industry standard. I'll suggest you remove this `disable` rule and try to fix the issue. :smile: :small_airplane: